packages/gn.rb

```diff
@@ -0,0 +1,30 @@
+require 'package'
+
+class Gn < Package
+  description 'GN is a meta-build system that generates build files for Ninja.'
+  homepage 'https://gn.googlesource.com/gn/'
+  version 'dca877f'
+  source_url 'https://chromium.googlesource.com/chromiumos/containers/sommelier/+/0.20/README?format=TEXT'
+  source_sha256 'b58d799b16d20abf92369fe0749c73f7398996f0afa9933517051778a8bb16c3'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+
+  def self.build
+    system "git clone https://gn.googlesource.com/gn"
+    system 'set C_INCLUDE_PATH /usr/local/include/'
```

Comment:
I don't believe this will work.  You will need to do something like:
```
env['C_INCLUDE_PATH'] = "#{CREW_PREFIX}/include"
```

Reply:
Absolutely right. But it also needs to set `CXX` to `g++`. So : a new dependency to gcc8

Reply:
You don't need to include `gcc8` as a dependency since it is already included in Chromebrew "core".  Like I said earlier, you can find the package(s) you don't need with `grep gcc8 /path/to/chromebrew/repo/tools/core_packages.txt`.  It should return the the same so you know "core" already includes that package.

Reply:
Right : It's late here and I can hardly keep my eyes open.